packages/cactus-plugin-bungee-hermes/src/main/typescript/view-creation/transaction-proof.ts

```diff
@@ -0,0 +1,35 @@
+import { Proof } from "./proof";
+
+export class TransactionProof {
+  private creator: Proof;
```

Comment:
Can we call `creator` class var as something different (Like how you have used the term `blockCreator`)? Because Proof itself has a creator class var